Source/Krypton Components/Krypton.Toolkit/Values/ColorButtonValues.cs

```diff
@@ -20,7 +20,7 @@ public class ColorButtonValues : Storage,
     {
         #region Static Fields
 
-        private string _defaultText = KryptonManager.Strings.GlobalColorStrings.Color;
+        private readonly string _defaultText = KryptonLanguageManager.ColorStrings.Color;
```

Comment:
Strings are now handled by `KryptonManager.Strings` (or at least in the `alpha` branch, as per PR #1166)

Reply:
Like I said the merge was mangled somehow !